Tests/DeepLearningTests/InitializerTests.swift

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import DeepLearning
+
+final class InitializerTests: XCTestCase {
+    func testInitializers() {
+        let scalar = Tensor<Float>(1)
+        let matrix: Tensor<Float> = [[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]
+        let broadcastScalar = Tensor<Float>(broadcasting: 10, rank: 3)
+        let some4d = Tensor<Float>(
+            shape: [2, 1, 2, 1],
+            scalars: AnyRandomAccessCollection([2, 3, 4, 5]))
```

Comment:
I don't think `AnyRandomAccessCollection` is necessary since `init(shape:scalars:)` is generic.
```suggestion
            scalars: [2, 3, 4, 5])
```